### PR TITLE
Bump version: 2.0.1 → 2.1.0: Add bucket_path convenience function

### PR DESCRIPTION
**Pull request by lgruen**
See https://centrepopgen.slack.com/archives/C01DNQ1E109/p1625107308176700 for context.

### CODE REVIEW

File: analysis_runner/__init__.py [hunk 1]
```diff
@@ -38,12 +42,51 @@ def output_path(filename: str, bucket_category: Optional[str] = None) -> str:
...
+    In contrast to the bucket_path function, output_path takes the "output" parameter
+    from the analysis-runner invocation into account.
+
+    Examples
+    --------
+
+    Assuming that the analysis-runner has been invoked with
+    `--dataset fewgenomes --access-level test --output 1kg_pca/v42`:
+
+    >>> from analysis_runner import output_path
```
Comment (illusional) [inline analysis_runner/__init__.py]: Do these examples get automatically tested?

**Review by illusional — APPROVED** [review]

File: analysis_runner/__init__.py (hunk 1 above)
Comment (illusional) [reply analysis_runner/__init__.py]: If not, would be cool at some point to add these doctests to the GH action.

Comment (lgruen) [reply analysis_runner/__init__.py]: Well, this particular one would rely on environment variables being set. Not sure if one could automate that easily?

Comment (illusional) [reply analysis_runner/__init__.py]: Ah true, not a further consideration then :)

**Review by KatalinaBobowik — APPROVED** [review]
Looks great!

**Pull request merged**